backend/tests/core/Entities/ProjectHelper.cs

```diff
@@ -79,8 +79,9 @@ public static List<Entity.Project> CreateProjects(int startId, int count, Entity
         /// <param name="id"></param>
         /// <param name="agency"></param>
         /// <returns></returns>
-        public static Entity.Project CreateProject(this PimsContext context, int id, Entity.Agency agency)
+        public static Entity.Project CreateProject(this PimsContext context, int id, int agencyId)
         {
+            var agency = EntityHelper.CreateAgency(agencyId);
```

Comment:
As this is a helper with `PimsContext`, you may want to first see if the agency exists.
`context.Agencies.Find(agencyId) ?? EntityHelper.CreateAgency(agencyId)`